### ISSUE TITLE
Top Interface Layers "brim" around first layer of model.

### ISSUE THREAD

**Issue by Deraj103** [bambulab/BambuStudio#2804]
### Bambu Studio Version

1.7.7.89

### Where is the application from?

Bambu Lab Official website

### OS version

Windows 10 Pro, 22H2

### Additional system information

CPU: Intel(R) Core(TM) i5-7600K CPU @ 3.80GHz   3.79 GHz
Memory: 32 GBs
GPU: NVIDIA GeForce RTX 3050

### Printer

Bambu Lab X1 Carbon

### How to reproduce

Adding a value to tree support brim width And adding top interface layers.

### Actual results

This creates a top interface layer "brim" around the points of contact of the model to the build plate. It seems to override the brims of the tree supports. Even to the point where the supports have no brim, causing them to be unstable and fall over. Increasing the tree support brim width increases the width of this top interface layer "brim".
![2023-10-29 21_33_38-](https://github.com/bambulab/BambuStudio/assets/125715452/0258a1ea-5d3f-4022-baa6-c3f5e6814a86)

### Expected results

If I remove the values of tree support brim width OR top interface layers, this odd brim goes away.

### Project file & Debug log uploads

[Summoned_Earth_Elemental.zip](https://github.com/bambulab/BambuStudio/files/13200490/Summoned_Earth_Elemental.zip)

### Checklist of files to include

- [ ] Log file
- [ ] Project file

**Comment by Deraj103** [comment]
Something I've noticed. Unsure if this is relevant. If I leave tree support brim width as 0, I noticed that there are a small amount of top interface layers on the first layer. This must be simply the design of the model. 
So why is the top interface layers "brim" influenced by the tree support brim width? Is it a legit feature or a bug?
I've attached a bottom view of the top interface layer "brim"
![underneath](https://github.com/bambulab/BambuStudio/assets/125715452/b639cce0-2d2e-45d7-be42-aa78bcd2e09e)
